circle.yml

```diff
@@ -0,0 +1,31 @@
+general:
+
+
+machine:
+  services:
+    - docker
+  environment:
+    GOPATH: /home/ubuntu:$GOPATH
+    SRCDIR: /home/ubuntu/src/github.com/weaveworks/weave
+    PATH: $PATH:$HOME/.local/bin:$HOME/google-cloud-sdk/bin/
+    CLOUDSDK_CORE_DISABLE_PROMPTS: 1
+
+dependencies:
+  post:
+    - docker build -t weaveworks/weave-build build
+    - pip install --user --upgrade gcloud gsutil
+    - curl https://sdk.cloud.google.com | bash
+    - bin/setup-circleci-secrets "$SECRET_PASSWORD"
+
+test:
+  pre:
+    - ls -al /var/run/docker.sock
+    - mkdir -p $(dirname $SRCDIR)
+    - cp -r $(pwd)/ $SRCDIR
+  override:
+    - docker run -v /var/run/docker.sock:/run/docker.sock -v /home/ubuntu:/home/go weaveworks/weave-build
+    - cd $SRCDIR/test; ./gce/gce.sh setup
+    - cd $SRCDIR/test; . ./gce/gce.sh hosts; ./setup.sh
```

Comment:
Don't you mean: ```. <(./gce/gce.sh hosts);``` here, and the line below? As, I don't think ```.``` will take multiple args.